veocean-data-farming/df-background.md

```diff
@@ -0,0 +1,55 @@
+---
+description: Intuition and background on the Data Farming program.
+---
+# Data Farming Background
+
+It rewards OCEAN to liquidity providers (stakers) as a function of consume volume and liquidity. It’s like DeFi liquidity mining, but tuned for data consumption. DF’s aim is to achieve a minimum supply of data for network effects to kick in, and once the network flywheel is spinning, to increase growth rate.
+
+
+## Active Work to Drive APY
+
+Data Farming is not a wholly passive activity. The name of the game is to drive Data Consume Volume (DCV). High APYs happen only when there is sufficiently high DCV. High DCV means publishing and consuming truly useful datasets (or algorithms).
+
+Thus, if you really want to max out your APY:
+- create & publish datasets (and make $ in selling them) — or work with people who can
+- consume the datasets (to make $) — or work with people who can
```

Comment:
"consume the datasets" sounds a bit like wash consume. I'd prefer if this said something like:

```suggestion
- Promote the consumption of the datasets — or work with people who can
```

Reply:
"promote the consumption" is too watered-down. 

And we want to emphasize making $.

How about "buy the datasets and use them to make $ - or work with people who can"

Reply:
Fixed CTA. Kept formatting consistent with the rest, made sentences uppercase.

```
Thus, if you really want to max out your APY:
- Create & publish datasets (and make $ in selling them) — or work with people who can
- Buy the datasets (and use them to make $) — or work with people who can
- Stake on them, and claim the rewards.
```